main.go

```diff
@@ -60,9 +71,9 @@ func startServer(zimService *zim.Handler) {
 }
 
 // Build a new Http server
-func createServer(router http.Handler) http.Server {
+func createServer(router http.Handler, port string) http.Server {
 	return http.Server{
-		Addr:    ":8080",
+		Addr:    port,
```

Comment:
I think you should rename `port` to address. ":8080" is just a shortcut for "0.0.0.0:8080" which is the full address where the server is going to listen on, so "address" as config and variable name would be more fitting.

Reply:
I named it `Port` because it seemed simple to understand for some users, but yes, it is indeed a good idea to change that.
Please check 31436ea and a4b146c. :)